crawler.py

```diff
@@ -93,46 +78,44 @@ def download_and_format(id: str) -> Study:
 
     #foramtting 
     title = data.find("official_title").get_text()
-    description = "" #TODO #summary?
-        
+    description = data.find("brief_summary").get_text()
+            
     date_str = data.find("last_update_posted").get_text()
     last_updated = date.strptime(date_str, '%B %d, %Y')
-        
+            
     study_type = data.find("study_type").get_text()
-    condition = [] #TODO 
-    sponsor = data.find("lead_sponsor").find("agency").get_text() ##lead sponsor and collborators
-    recruitment_status = data.find("overall_status").get_text()
-        
-    min_age = data.find("minimum_age").get_text()
-    max_age = data.find("maximum_age").get_text()
-    unit = min_age[min_age.index(" ")+1:]
-    age = (int("".join(filter(str.isdigit, min_age))), 
-            int("".join(filter(str.isdigit, max_age))), unit)
+
+    conditions = []
+    for condition in data.findAll("condition"):
+        conditions.append(condition)
         
+    sponsor = data.find("lead_sponsor").find("agency").get_text() ##lead sponsor only (no collborators)
+    recruitment_status = data.find("overall_status").get_text()
+            
+    age = {"min":data.find("minimum_age").get_text(), "max":data.find("maximum_age").get_text()}
+            
     sex = data.find("gender").get_text()
-    control = "" #TODO #??
-    additional_criteria = "" #TODO #inclusion/exclusion
+    control = data.find("healthy_volunteers").get_text()
```

Comment:
This part caused an error when I tested your code on  random studies. I recommend doing this


```suggestion
try:
       control = data.find("healthy_volunteers").get_text()
except AttributeError:
       control = None
```